### ISSUE TITLE
Speedy compat issue

### ISSUE THREAD

**Issue by bdbxml** [BobbyCannon/Bloodhound#1]
Upon updating the NuGet packages in the Scribe wiki, running into issues where Bloodhound isn't compatible with the current version of Speedy. 

Great libraries btw!

**Comment by BobbyCannon** [comment]
Yeah, I have not been maintaining Bloodhound and Scribe due to other priorities. I'll take a few and look it. I'll see if I can roll a new version to get all the dependencies back in line. I'll post back here when it's done. I should have some time too look at it tomorrow.

**Comment by BobbyCannon** [comment]
OK, I did a quick look. I'm going to need to spend a bit more time and refresh / update these projects.
